protocol/communities/manager.go

```diff
@@ -3623,3 +3900,13 @@ func revealedAccountsToAccountsAndChainIDsCombination(revealedAccounts []*protob
 	}
 	return accountsAndChainIDs
 }
+
+func (m *Manager) isWaleltsHasAdminPermission(becomeAdminPermissions []*protobuf.CommunityTokenPermission, accounts []*AccountChainIDsCombination) bool {
```

Comment:
```suggestion
func (m *Manager) walletHasAdminPermission(becomeAdminPermissions []*protobuf.CommunityTokenPermission, accounts []*AccountChainIDsCombination) bool {
```

Also, as an FYI, I've introduced `checkPermissions()` which is essentially the same as `checkPermissionToJoin()` because functionality-wise, they do exactly the same thing, as you've probably figured out here as well..

https://github.com/status-im/status-go/pull/3605/files#diff-0c22647108517d38489ae5faa65a914ade3be02ea7a7e37355a82739da9aeb69R1640-R1641